engine/src/feedback/feedback-context-info.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const atob = require('atob')
+const btoa = require('btoa')
+const RelativeUrl = require('./relative-url')
+
+class FeedbackContextInfo {
+  constructor (formTitle, pageTitle, url) {
+    this.formTitle = formTitle
+    this.pageTitle = pageTitle
+    // parse as a relative Url to ensure they're sensible values and prevent phishing
+    this.url = url ? new RelativeUrl(url).toString() : url
+  }
+
+  toString () {
+    return btoa(JSON.stringify(this))
+  }
+}
+module.exports.FeedbackContextInfo = FeedbackContextInfo
+
+module.exports.decode = function decode (encoded) {
```

Comment:
I lifted it from the hof library I wrote to do the same thing. I was using module.exports as that seemed to be consistent with what other things in engine were doing (e.g. address-service, index, model and page. Should we be preferring the export class.... type syntax, I thought this was more consistent with nodejs which the files in here seem to be?

Reply:
np, fine as is. previously the repo didn't have transpilation, so it was using the cjs syntax (module.exports). since we have babel it should transpile `export TheClass..` to `module.exports.TheClass`